contrib/dhcp_agent/dhcp_agent.py

```diff
@@ -0,0 +1,399 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import atexit
+import errno
+import fcntl
+import hashlib
+import logging
+import os
+import subprocess
+import sys
+import tempfile
+from optparse import OptionParser
+from logging import handlers as logging_handlers
+
+IS_PY3 = sys.version_info[0] == 3
+if IS_PY3:
+    from dbm import gnu as gdbm
+    from urllib.request import urlopen, Request
+    from urllib.parse import urlencode
+    from urllib.error import HTTPError
+else:
+    import gdbm
+    from urllib import urlencode
+    from urllib2 import urlopen, Request, HTTPError
+
+
+MODES = ('all', 'networks', 'entries')
```

Comment:
Don't you think that `MODES` is kind of ambiguous here..? (`MODES` of what..? maybe `EDIT_MODES`..?)